airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -137,6 +137,20 @@ def run_job_group(self, body_request: dict) -> dict[str, Any]:
         self._raise_for_status(response)
         return response.json()
 
+    @retry(stop=stop_after_attempt(5), wait=wait_exponential(multiplier=1, max=10))
+    def create_flow(self, *, body_request: dict) -> dict:
+        """
+        Creates flow.
+
+        :param body_request: Body of the POST request to be sent.
+            For more details check https://clouddataprep.com/documentation/api#operation/createFlow
```

Comment:
what is this link? is this google domain?
We must use only official links
I know links are by `https://cloud.google.com/dataprep`

Reply:
Dataprep by Trifacta is a service integrated with GCP but lives beyond it. That's why there's Trifacta's official documentation is used here, and Google's page `https://cloud.google.com/dataprep` points to Trifacta's resources.